documentation/sphinx/source/configuration.rst

```diff
@@ -215,8 +213,25 @@ Contains basic configuration parameters of the ``fdbmonitor`` process. ``restart
 
     [general]
     cluster_file = /etc/foundationdb/fdb.cluster
-
-Contains settings applicable to all processes (e.g. fdbserver, backup_agent). The main setting of interest is ``cluster_file``, which specifies the location of the cluster file. This file and the directory that contains it must be writable by all processes (i.e. by the user or group set in the [fdbmonitor] section).
+    restart_delay = 60
+    ## by default, restart_backoff = restart_delay_reset_interval = restart_delay
+    # initial_restart_delay = 0
+    # restart_backoff = 60.0
+    # restart_delay_reset_interval = 60
+    # delete_envvars =
+    # kill_on_configuration_change = true
+    # disable_lifecycle_logging = false
+
+Contains settings applicable to all processes (e.g. fdbserver, backup_agent).
+
+* ``cluster_file``: Specifies the location of the cluster file. This file and the directory that contains it must be writable by all processes (i.e. by the user or group set in the ``[fdbmonitor]`` section).
+* ``restart_delay``: The restart delay parameters control how long ``fdbmonitor`` waits to restart a process when it dies. ``fdbmonitor`` uses backoff logic to prevent a process that dies repeatedly from cycling too quickly, and it also introduces up to +/-10% random jitter into the delay to avoid multiple processes all repeatedly starting simultaneously. The ``restart_delay`` is the maximum value in seconds for the delay, and by default is also the value used for ``restart_backoff`` and ``restart_delay_reset_interval``. This means that if other backoff parameters are not set, the second and subsequent restarts will all take ``restart_delay`` seconds, and the backoff will reset after a process has been running for ``restart_delay`` seconds.
```

Comment:
You might wish to move the first half of this paragraph into a small paragraph over all of the restarting parameters to make the grouping more obvious?

Reply:
I'll try to reorganize this a bit, which may make it easier to also incorporate Markus's suggestion.